### PR TITLE
Fix #29

### LINKED ISSUE #29: Find a way not to pass CSRF token around
- https://github.com/yiisoft/app/blob/master/src/Controller/SiteController.php#L18 - explicitly passing CSRF token from request to view.
- https://github.com/yiisoft/app/blob/master/resources/layout/main.php#L29 - passing CSRF around.
- https://github.com/yiisoft/app/blob/master/resources/views/contact/form.php#L26 - explicitly using token.

That looks a bit too verbose and likely affects overall usability of the framework. Is there a way to avoid it?

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #29 

**Can be merged after yiisoft/view#86**

### CODE REVIEW

**Comment by terabytesoftw** [comment]
Why remove meta csrf _head ?

**Comment by rustamwin** [comment]
>Why remove meta csrf _head ?

No longer needed since View will have a `csrf` method.

**Comment by samdark** [comment]
Decided to do it the same way as in demo: https://github.com/yiisoft/yii-demo/commit/9f608a48f773aa1215c19b32b0ddc0ce430dc8f7

File: resources/layout/_head.php [hunk 1]
```diff
@@ -14,9 +14,6 @@
     <meta charset="<?= Html::encode($applicationParameters->getCharset()) ?>">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <?php if ($csrf !== null): ?>
-        <meta name="csrf" content="<?= $csrf ?>">
```
[the remark this answers is not in the document]
Comment (samdark) [reply resources/layout/_head.php:18]: https://github.com/yiisoft/app/pull/34/files#diff-01449f17669b0498ebd16c8e2d150dd2R124

**Comment by samdark** [comment]
👍

**Pull request merged**